### DISCUSSION TITLE
simGetCollisionInfo().has_collided always false, plz help

### DISCUSSION THREAD

**Discussion by zsychina** [microsoft/AirSim#4924] [General]
I'm working on a collision avoidance algorithm using AirSim for reinforcement learning and I've encountered an issue where simGetCollisionInfo().has_collided is not returning whether a collision has actually happened. The has_collided flag remains false even though my drone has already collided with a block.

The same is true in the UE interface; the system has already displayed the nth collision, but the collision_count remains at 0.

I'm new to this area, so any response would be helpful.
![Snipaste_2024-02-02_22-49-51](https://github.com/microsoft/AirSim/assets/89634324/40323f7e-2a50-4cd9-ba58-3811a733b03d)

**Comment by KeepOnCarvingHuang** [comment]
I also have the same problem, have you solved it?

**Reply by zsychina** [reply]
Hi Huang, I used an alternative solution by adding 4 distant sensors around the drone, lol
Sry for my late reply.
